yellowbrick/classifier/threshold.py

```diff
@@ -0,0 +1,351 @@
+        Returns the axes that the parallel coordinates were drawn on.
+    """
+    # Instantiate the visualizer
+    visualizer = ThresholdVisualizer(
+        model,
+        colormap=colormap,
+        color=color,
+        n_trials=n_trials,
+        test_size_percent=test_size_percent,
+        quantiles=quantiles,
+        random_state=random_state,
+        title=title,
+        **kwargs)
+
+    # Fit and transform the visualizer (calls draw)
+    visualizer.fit_draw_poof(X, y)
+
+    # Return the axes object on the visualizer
+    return visualizer.ax
+
+
+##########################################################################
+# Static ThresholdVisualizer Visualizer
+##########################################################################
+
+
+class ThresholdVisualizer(ModelVisualizer):
+    """Visualizes precision, recall and queue rate at different different
+    thresholds for binary targets.
+
+    Parameters
+    ----------
+
+    model : a Scikit-Learn classifier, required
+        Should be an instance of a classifier otherwise a will raise a
+        YellowbrickTypeError exception on instantiation.
+
+    colormap : string or cmap, default: None
+        Optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    color : String, default: None
+        Optional string or matplotlib cmap to colorize lines
+        Use either color to colorize the lines on a per class basis or
+        colormap to color them on a continuous scale.
+
+    n_trials : Integer, default: 50
+        Number of trials to conduct via train_test_split
+
+    quantiles : Sequence, default: (0.1, 0.5, .9)
+        Setting the quantiles for visualizing model variability using
+        scipy.stats.mstats.mquantiles
+
+    random_state : Integer, default: 0
+        Random state integer for sampling in train_test_split
+
+    title : String, default: None
+        Title of the visualization
+
+    kwargs : keyword arguments passed to the super class.
+    """
+
+    def __init__(self,
+                 model,
+                 colormap=None,
+                 color=None,
+                 n_trials=50,
+                 test_size_percent=0.1,
+                 quantiles=(0.1, 0.5, 0.9),
+                 random_state=0,
+                 title=None,
+                 **kwargs):
+        # Check to see if model is an instance of a classifier.
+        # Should return an error if it isn't.
+        if not isclassifier(model):
+            raise YellowbrickTypeError(
+                "This estimator is not a classifier; try a regression or clustering score visualizer instead!"
+            )
+        super(ThresholdVisualizer, self).__init__(model, **kwargs)
+
+        self.estimator = model
+        self.colormap = kwargs.pop('colormap', None)
+        self.color = kwargs.pop('color', None)
+        self.title = title
```

Comment:
Note that the super class, `Visualizer` handles title and implements `set_title()`. [yellowbrick.base L71](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/base.py#L71)